libraries/chain/include/eosio/chain/controller.hpp

```diff
@@ -270,6 +270,7 @@ namespace eosio::chain {
          // Called by qc_chain to indicate the current irreversible block num
          // After hotstuff is activated, this should be called on startup by qc_chain
          void set_if_irreversible_block_num(uint32_t block_num);
+         uint32_t if_lib_num() const;
```

Comment:
Should match the name of the `set`. `if_irreversible_block_num()`

Reply:
Thanks. Initially I used `if_irreversible_block_num` and then thought it was too long. I will change back to `if_irreversible_block_num` to be consistent.